data_handler/handlers/order_books/myswap/main.py

```diff
@@ -0,0 +1,240 @@
+import asyncio
+import math
+from decimal import Decimal
+from pathlib import Path
+
+from handlers.blockchain_call import func_call
+from handlers.order_books.abstractions import OrderBookBase
+
+import pandas as pd
+
+from handlers.order_books.commons import get_logger
+from handlers.order_books.myswap.api_connection.api_connector import MySwapAPIConnector
+from handlers.order_books.myswap.api_connection.data_collectors import braavos_get_tokens_prices
+
+
+MYSWAP_CL_MM_ADDRESS = "0x01114c7103e12c2b2ecbd3a2472ba9c48ddcbf702b1c242dd570057e26212111"
+
+# The maximum tick value available in liqmap.json.gz from MySwap Data Service
+MAX_MYSWAP_TICK = Decimal("1774532")
+
+
+class MySwapOrderBook(OrderBookBase):
+    """Class for MySwap order book."""
+    DEX = "MySwap"
+    MYSWAP_URL = "https://myswap-cl-charts.s3.amazonaws.com/data/pools/{pool_id}/liqmap.json.gz"
+
+    def __init__(self, token_a: str, token_b: str, apply_filtering: bool = False):
+        """
+        Initialize the MySwap order book.
+        :param token_a: str - The base token address in hexadecimal.
+        :param token_b: str - The quote token address in hexadecimal.
+        :param apply_filtering: bool - Apply filtering to the order book.
+        """
+        super().__init__(token_a, token_b)
+        self.connector = MySwapAPIConnector()
+        self.apply_filtering = apply_filtering
+        self.logger = get_logger("MySwap", Path().resolve().joinpath("./logs"))
+        self.token_a_name, self.token_b_name = map(lambda info: info.name, self.get_token_configs())
+        self._usd_price = Decimal("0")
+        self._decimals_diff = Decimal(10 ** (self.token_a_decimal - self.token_b_decimal))
+        self._set_usd_price()
```

Comment:
This sets the USD price for the base token. But don't we need the price for the quote token as well?